blazor/speeddial/display-modes.md

```diff
@@ -7,17 +7,66 @@ control: SpeedDial
 documentation: ug
 ---
 
-# Display Modes in Blazor Speed Dial Component
+# Display Options in Blazor SpeedDial Component
 
 The action items in Blazor Speed Dial can be displayed in `Linear` and `Radial` display modes by setting [Mode](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_Mode) property.
 
-## Linear display mode
+## Open trigger
+
+By default, action items are displayed only when clicking the speed dial button. You can use the [OpensOnHover](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Buttons.SfSpeedDial.html#Syncfusion_Blazor_Buttons_SfSpeedDial_OpensOnHover) property to change the trigger that opens the action items to be on hover instead.
```

Comment:
By default, action items are displayed only on clicking the speed dial button. You can use the [OpensOnHover]() property to open the action items by hovering the speed dial button.

Reply:
Modified the content as given